README.md

```diff
@@ -1,4 +1,4 @@
-# Awesome Bioinformatics (WUBSyB Version)
+﻿# Awesome Bioinformatics (WUBSyB Version)
```

Comment:
The difference in this line is that a [BOM](https://en.wikipedia.org/wiki/Byte_order_mark) has been added. This was probably done automatically by the text editor. Just out of curiosity: which editor do you use @Joerg-Schultz ?
The BOM is optional and does not hurt, so in my opinion we can keep it.